tests/unit_tests/test_config.py

```
@@ -1,32 +1,43 @@
+def test_timstof():
+    config = Config("timstof")
+    assert config.random_seed == 454
+    assert config._config_file.name == "config_timstof.yaml"
+    assert config.file == "timstof"
+
+    config = Config("tims")
+    assert config.random_seed == 454
+    assert config._config_file.name == "config_timstof.yaml"
+    assert config.file == "tims"
+
+    with pytest.raises(ValueError):
```

Ruff (0.15.15)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)